env0/resource_environment.go

```diff
@@ -650,6 +665,24 @@ func updateTemplate(d *schema.ResourceData, apiClient client.ApiClientInterface)
 	return nil
 }
 
+func updateDriftDetection(d *schema.ResourceData, apiClient client.ApiClientInterface) diag.Diagnostics {
+	drift_detection_cron, ok := d.GetOk("drift_detection_cron")
+	if !ok || drift_detection_cron.(string) == "" {
+		if err := apiClient.EnvironmentStopDriftDetection(d.Id()); err != nil {
```

Comment:
for updates there's stop drift API and update drift API.
No option to update directly from the environment API.

Reply:
I think that it was fine not supporting update or delete altogether because the UI also doesn't support it, but since this was implemented already and may provide better UX - let's keep it 😄